Tests/NIOSOCKSTests/ClientRequest+Tests.swift

```diff
@@ -58,13 +58,13 @@ extension ClientRequestTests {
     func testWriteAddressType(){
         var ipv4 = ByteBuffer()
         XCTAssertEqual(ipv4.writeAddressType(.address(try! .init(ipAddress: "192.168.1.1", port: 80))), 7)
-        XCTAssertEqual(ipv4.readBytes(length: 5)!, [1, 1, 1, 168, 192])
-        XCTAssertEqual(ipv4.readInteger(as: UInt16.self)!, 0x5000)
+        XCTAssertEqual(ipv4.readBytes(length: 5)!, [1, 192, 168, 1, 1])
+        XCTAssertEqual(ipv4.readInteger(as: UInt16.self)!, 0x0050)
```

Comment:
Might as well make it more obvious:
```suggestion
        XCTAssertEqual(ipv4.readInteger(as: UInt16.self)!, 80)
```